patches/third_party-rust-serde-v1-BUILD.gn.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/third_party/rust/serde/v1/BUILD.gn b/third_party/rust/serde/v1/BUILD.gn
+index 0cc7bd91f39cd6cff8349a73791d04e6cca24509..498d55c57824a02e7bed15eee0f2c00d6e48bf26 100644
+--- a/third_party/rust/serde/v1/BUILD.gn
++++ b/third_party/rust/serde/v1/BUILD.gn
+@@ -30,4 +30,5 @@ cargo_crate("lib") {
+   ]
+   build_root = "crate/build.rs"
+   build_sources = [ "crate/build.rs" ]
++  features += [ "rc" ]
```

Comment:
These are all features that we have enabled, but were not in upstream. I need to double check that they're all actually needed (along with double-checking all features in third party rust deps)

Reply:
I looked briefly and wasn't able to trace this one; it's not in our direct code. I filed https://github.com/brave/brave-browser/issues/31317 as a follow up.

Reply:
rc for adblock and alloc for serde_ipld_dagcbor